k8s/namespaces/bsp/bulk-scan-orchestrator/aat.yaml

```diff
@@ -37,6 +41,11 @@ spec:
           CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
           IDAM_API_URL: "https://idam-api.aat.platform.hmcts.net"
           IDAM_CLIENT_REDIRECT_URI: "https://bulk-scan-orchestrator-aat.service.core-compute-aat.internal/oauth2/callback"
+          QUEUE_NAMESPACE = "bulk-scan-servicebus-aat"
+          QUEUE_ENVELOPES_NAME = "envelopes"
+          QUEUE_PAYMENTS_NAME = "payments"
+          QUEUE_READ_ACCESS_KEY_NAME = 'ListenSharedAccessKey'
+          QUEUE_WRITE_ACCESS_KEY_NAME = 'SendSharedAccessKey'
```

Comment:
should be colons :)

Reply:
yea, different language source i copied those. kustomize tests saved me and fixed it :innocent: